src/apps/AppManager.jsx

```diff
@@ -98,7 +98,7 @@ export default class AppManager {
                 display: 'Compass',
                 app: CoreCancerPilotApp,
                 isExact: true,
-                dataSource: 'HardCodedReadOnlyDataSource',
```

Comment:
If the HardCodedReadOnlyDataSource is replaced everywhere, is there any reason not to just delete it altogether? It references the same patient jsons as the McodeV01DataSource, but doesn't wrap them in EntryMapper.mapEntries so if it does still get used somewhere it probably isn't going to work.  I notice that it is still referenced as a fallback in case the datasource is not defined in props in FullApp:
```
        if (Lang.isUndefined(this.props.dataSource)) {
            this.dataAccess = new DataAccess("HardCodedReadOnlyDataSource");
        } else {
            this.dataAccess = new DataAccess(this.props.dataSource);
        }
```

Reply:
Good catch, I'll delete this.